opentreemap/treemap/views/misc.py

```diff
@@ -117,6 +118,18 @@ def boundary_to_geojson(request, instance, boundary_id):
     return HttpResponse(geom.geojson)
 
 
+def add_anonymous_boundary(request):
+    request_dict = json.loads(request.body)
+    polygon = Polygon(request_dict.get('polygon', []))
+    b = Boundary()
+    b.geom = MultiPolygon(polygon)
+    b.name = ''
+    b.category = ''
+    b.sort_order = 1
+    b.save()
```

Comment:
The `boundary_autocomple` method in `treemap/views/misc.py` filters based on the value of the `searchable` attribute, which defaults to `True`.

https://github.com/OpenTreeMap/otm-core/blob/caa9153a73cbebb1518a522bbb38fcba8727e252/opentreemap/treemap/views/misc.py#L124

We should set `searchable` to `False` for anonymous boundaries.

I would also like to discuss possibly using a custom manager to keep the logic that defines what "anonymous" means together in the model layer.

[NOTE: this is speculative, untested code]

```python
class BoundaryManager(models.GeoManager):
  def create_anonymous_boundary(self, coordinates):
    polygon = Polygon(request_dict.get('polygon', []))
    return self.create(geom=MultiPolygon(polygon),
                       name='',
                       category='',
                       sort_order = 1,
                       searchable=False)

  def anonymous(self):
    return self.get_queryset().filter(name='', category='')

  def named(self):
    return self.get_queryset().exclude(name='', category='')
```